sirepo/uri_router.py

```diff
@@ -212,7 +212,7 @@ def _dispatch(path):
                     raise sirepo.util.raise_not_found('{}: uri missing parameter ({})', path, p.name)
                 break
             if p.is_path_info:
-                kwargs[p.name] = '/'.join(parts)
+                kwargs[p.name] = '/'.join([x.lstrip(_PATH_INFO_CHAR) for x in parts])
```

Comment:
I don't understand this change. Why would parts contain _PATH_INFO_CHAR?

Reply:
@robnagler the [GUI doesn't strip the _PATH_INFO_CHAR](https://github.com/radiasoft/sirepo/blob/1912882dc322db1f33962e87148aeddc9c6ef2d7/sirepo/package_data/static/js/sirepo.js#L1821) from the url when it does a redirect. So it is sent to the server. It needs to be stripped so we can find the simulation type (ex *elegant -> elegant). Should the GUI strip the char before sending? This will also solve my comment above:
>@robnagler one possible issue: the URLS are formatted like v.radia.run:8000/*elegant in the address bar. Is this what you want or should the * be removed?

Reply:
Yes, that's a bug in the GUI.

Reply:
https://github.com/radiasoft/sirepo/pull/2786/commits/d8ea1bab46d53e05ed40c59d87e3acd76912f310